### PR TITLE
Add documentation to ImageReconstruction

### PR DESCRIPTION
**Pull request by grey277**
Add documentation to ImageReconstruction

### CODE REVIEW

File: ImageReconstruction/PARAMETERS.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+- `SinogramCreatorMC_OutFileName_std::string`
+  Path to file where sinogram will be saved.
+
+- `SinogramCreatorMC_ReconstructionDistanceAccuracy_float`
+  Used to decrease size of bin, if reconstruction distance accuracy is 1: 1 bin correspondes to 1cm
+
+- `SinogramCreatorMC_SinogramZSplitNumber_int`
+  Number of sinograms along Z axis. Z axis is divided by this number and then evenly split.
+
+- `SinogramCreatorMC_ScintillatorLenght_float`
```
Comment (wkrzemien) [inline ImageReconstruction/PARAMETERS.md]: it is duplicated I think

Comment (grey277) [reply ImageReconstruction/PARAMETERS.md]: I do not see any duplicates, there are SinogramCreator and SinogramCreatorMC, 2 different user tasks with different userparams, but most of their userparams have same meaning.

**Comment by grey277** [comment]
I do not see any duplicates, there are SinogramCreator and SinogramCreatorMC, 2 different user tasks with different userparams, but most of their userparams have same meaning.

File: ImageReconstruction/PARAMETERS.md (hunk 1 above)
Comment (wkrzemien) [reply ImageReconstruction/PARAMETERS.md]: Sorry. My mistake

**Pull request merged**